components/shared/CustomButton.tsx

```diff
@@ -54,7 +53,7 @@ const CustomButton = React.forwardRef<HTMLButtonElement, CustomButtonProps>(
       <Button
         ref={ref}
         className={`${button.className} ${className}`}
-        {...rest}
+        {...props}
```

Comment:
Not sure why we reverted this change.

![image](https://github.com/agordon123/git_note/assets/4108484/e23fa813-ce0a-4027-b9b9-dde475a62252)